src/main/java/no/digipost/print/validate/PdfValidationError.java

```diff
@@ -34,7 +34,8 @@
     PDF_PARSE_ERROR("Could not parse the PDF document."),
     PDF_PARSE_PAGE_ERROR("Could not parse at least one of the pages in the PDF document"),
     UNSUPPORTED_DIMENSIONS("The dimensions of the PDF document are not supported. Supported dimensions are A4 (" + PdfValidator.A4_WIDTH_MM + " mm x "
-            + PdfValidator.A4_HEIGHT_MM + " mm). For flexibility, we allow smaller sizes down to a limit of " + PdfValidator.ALLOWED_NEGATIVE_DEVIATION_FROM_PDF_STANDARD_DIMENSIONS_MM + " mm for both dimensions"),
+            + PdfValidator.A4_HEIGHT_MM + " mm). For flexibility, we allow smaller sizes down to a limit of %s mm for both dimensions and larger sizes up to a limit of %s for both dimensions. " +
```

Comment:
Kanskje vi burde droppe den setningen med visbility og bare ha "Supported dimensions are width between `PdfValidator.A4_WIDTH_MM - negativeBleed`and `PDfValidator.A4_WIDTH_MM + positive bleed` and height between .....

Hva tror du?